sdk-core/smart_contract_query.md

```diff
@@ -13,6 +13,7 @@ dto SmartContractQuery:
 
 ```
 dto SmartContractQueryResponse:
+    original_query?: SmartContractQuery;
```

Comment:
Necessary to recover the function called by the query, to enable ABI-informed decoding.

Question for review: or, perhaps, only hold `original_query_function`?

Reply:
Changed, to hold `original_query_function`.